src/components/30-organisms/testimonials/story.js

```diff
@@ -45,25 +45,28 @@ story.add('Testimonials', () => {
         >
         <span class="author">Andrew Jackson, Advocate</span>
       </div>
-      <div>
+      <div class="showallinline_ie_bugfix">
```

Comment:
As discusses, this shouldn't have a name like this. @LucaMele  Maybe you can take a look at the bug dani is workarounding here.

Reply:
dtto, name does not follow our css rules: `o-testimonials__showallinline__ie__bugfix`